### ISSUE TITLE
Couldn't connect to the Illustrator plugin error for AI 2022

### ISSUE THREAD

**Issue by eduard-buta** [WestonThayer/Bloks#19]
Hi. Unfortunately, I can't get this to work. It gives me this error: Uhoh, we couldn't connect to the Illustrator plugin. Make sure BloksAIPlugin.aip is installed correctly.

I've made sure to follow your instructions, but without any luck. Looking forward to hearing your thoughts on this matter!

**Comment by jctremblay65** [comment]
I have the same issues...
AI 26.2.1 - MacOS 12.2.1 - Apple M1 Max

<img width="292" alt="Screen 2" src="https://user-images.githubusercontent.com/56195857/163598908-47c0936e-317e-4dc1-934f-27f9fdbd2303.png">

**Comment by igormedina** [comment]
Same issue Ai 26.2.1 - MacOS 11.6.5 - intel Core i7

**Comment by WestonThayer** [comment]
Probably due to merging https://github.com/WestonThayer/Bloks/pull/18. Sorry I didn't test it (don't even have a Creative Cloud subscription anymore)! Very cool that y'all are using Bloks, didn't realize people did.

Does it work if you uninstall and use the old release? https://github.com/WestonThayer/Bloks/releases/tag/v0.1.1

**Comment by eduard-buta** [comment]
> Probably due to merging #18. Sorry I didn't test it (don't even have a Creative Cloud subscription anymore)! Very cool that y'all are using Bloks, didn't realize people did.
> 
> Does it work if you uninstall and use the old release? https://github.com/WestonThayer/Bloks/releases/tag/v0.1.1

Hi Weston. Great hearing from you here. Unfortunately, the old version doesn't work either. Not for me, at least. It's a shame too since this is a plugin that had the potential of being a game-changer for those of us who work with layouts in Illustrator. Is there any chance for future support?

**Comment by WestonThayer** [comment]
That's super cool, I'm a big believer in layout for design tools, even if they aren't specifically for UI design (Figma, XD).

I don't have any time to support this, currently running a startup. I started this when working at Microsoft because they were stuck using Illustrator for everything, but I think they switched to Figma. If someone else wants to take over, that's fine with me, I can point to a fork.

If you have resources to fund some work on it (would probably be very minimal upgrade work to the latest SDKs, but you never know), I remember https://twitter.com/undavide was how I figured out half of this stuff. You could try paying him, or maybe he could refer you.

**Comment by sm1therz** [comment]
Bump! Any progress on this? I'd be willing to pitch in to have this fixed.

**Comment by WestonThayer** [comment]
No, I haven't seen any PRs. More than happy to accept them.

**Comment by sm1therz** [comment]
PR? Sorry. I feel really old right now.

**Comment by WestonThayer** [comment]
Oh no worries, PR = Pull Request. It's GitHub's convention for contributing changes to a code repository like this one https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests

**Comment by sm1therz** [comment]
Ah, thanks for that.  I'm asking around. People who've dev'ed other scripts/plugins would be best suited for updating this right?

**Comment by WestonThayer** [comment]
Yeah, probably. Illustrator's developer documentation isn't exactly well maintained or easy to understand (or at least it wasn't when I was working on this).

**Comment by oneezy** [comment]
was really hoping to use this! i haven't tested but looks like there's still issues. bummer.

EDIT: tested, still broke :(

**Comment by oneezy** [comment]
@wcork , any idea what might be wrong? looks like you were the last to update
